docker-compose.seagull.yaml

```diff
@@ -97,6 +106,25 @@ services:
       - DATASTORE_DATASET=${GCP_PROJECT_ID}
     command: ["gcloud", "beta", "emulators", "datastore", "start", "--host-port=0.0.0.0:8081"]
 
+  ext-auth-provider:
+    container_name: ext-auth-provider
+    build: ext-auth-provider
+
+  smdp-plus-emulator:
+    container_name: smdp-plus-emulator
+    build: sim-administration/sm-dp-plus-emulator
+    ports:
+      - "18080:8080"
+      - "18081:8081"
+
+  postgres:
+    container_name: postgres
+    build:
+      context: sim-administration/postgres
+      dockerfile: Dockerfile
+    tmpfs: "//var/lib/postgresql/data"
```

Comment:
double slash?   (just a nit, but is it significant in any way, if not then it would be clearer to just use a single slash, and if it is significant, add a comment).

Reply:
fixed